lib/cc/importer/canvas/module_converter.rb

```diff
@@ -49,6 +49,8 @@ def convert_modules(doc)
           item[:linked_resource_id] = get_node_val(item_node, 'identifierref')
           item[:linked_resource_global_id] = get_node_val(item_node, 'global_identifierref')
 
+          item[:part_id] = get_node_val(item_node, 'part_id')
```

Comment:
Is this part of the magic field fix?  Or partial work from the dynamic syllabus work?